_partials/windows_postgresql.md

```diff
@@ -18,6 +18,13 @@ sudo /etc/init.d/postgresql start
 sudo -u postgres psql --command "CREATE ROLE \"`whoami`\" LOGIN createdb superuser;"
 ```
 
+<details>
+
+If you get an error saying `could not change directory to "/home/your_name": Permission denied` that is fine.
+If on the first run it says `CREATE ROLE` in the end all worked fine.
+
+</details>
```

Comment:
```suggestion
_📝 If you get an error saying `could not change directory to "/home/your_name": Permission denied` that is totally fine, and there's no need to worry 💆 If, on the first run, it says `CREATE ROLE` in the end all worked fine._
```

Putting it in a `<details>` tag without a `<summary>` tag would create a disclosure triangle with no title, so people wouldn't open it. In general, students have found `<details><summary>` confusing and don't know how to open them (I can't believe I'm saying this, but it's the truth). So, personally I prefer italicized text.